### PR TITLE
Adapt Makefile rules for py3 changes

### PR DESCRIPTION
**Pull request by luizirber**
# WIP

Makefile rules were outdated, trying to fix it. (see inline comments)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,9 +39,9 @@ install-dependencies:
 	pip install --upgrade $(DEVPKGS)
 
 ## sharedobj   : build khmer shared object file
-sharedobj: khmer/_khmermodule.so
+sharedobj: khmer/_khmer.so
```
Comment (luizirber) [inline Makefile]: This is not true for py3, since it creates the dynamic library as `_khmer.cpython-34m.so`, for example.

File: Makefile [hunk 2]
```diff
@@ -154,7 +154,7 @@ diff_pylint_report: pylint_report.txt
 # We need to get coverage to look at our scripts. Since they aren't in a
 # python module we can't tell nosetests to look for them (via an import
 # statement). So we run nose inside of coverage.
-.coverage: $(PYSOURCES) $(wildcard tests/*.py) khmer/_khmermodule.so
+.coverage: $(PYSOURCES) $(wildcard tests/*.py) $(wildcard khmer/_khmer*.so)
```
Comment (luizirber) [inline Makefile]: This can be improved. We can create separate coverage runs and combine the results for the complete coverage.

**Comment by luizirber** [comment]
@mr-c ideas?

**Comment by mr-c** [comment]
Make use of http://stackoverflow.com/questions/4933285/how-to-detemine-python-version-in-makefile ?

**Comment by mr-c** [comment]
@luizirber ping?

**Comment by luizirber** [comment]
@mr-c I will add these changes to #1108

**Pull request closed without merging**